.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: release
+
+on:
+  workflow_call:
+    inputs:
+      readme:
```

Comment:
* Can you find a better name for these inputs?
* Can you add a description what these inputs do?
* You can use options to define what can be inserted.
* 
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
hmm, the linked blog states, that these options are for manually triggered workflows. We can try to check if this also works for workflow calls.
Your comment made me think again. Maybe we should drop the parameters and instead create separate fixed workflows. We maybe have 2-3 different release workflows. That is an amount where copying stuff still seems ok.

Reply:
Maybe something like this ?

```on:
  workflow_call:
    inputs:
      readme_docs:
        required: false
        type: string
        description: creates a README over a tool based Github action
        options:
        - terraform
      commit_files:
        required: false
        type: string
        description: specific files which should be added during the release build e.g. README.md, see https://github.com/github-actions-x/commit/blob/master/action.yml - files option
```

example for usage can be found here https://github.com/T-Systems-MMS/terraform-template/blob/main/.github/workflows/release.yml

Reply:
I'll take a looku to make seperate release.yml for terraform